.storybook/utils.ts

```diff
@@ -14,15 +14,26 @@ import {
 } from "@storybook/addon-knobs";
 
 import colors from "../node_modules/@esri/calcite-colors/colors.json";
-import { DocsPage } from "@storybook/addon-docs/blocks";
 
-export const darkBackground = [
-  {
-    name: "Dark",
-    value: colors["blk-210"],
-    default: true
-  }
-];
+export const lightBackground = {
+  default: "Light",
+  values: [
+    {
+      name: "Light",
+      value: "#f8f8f8"
+    }
+  ]
+};
+
+export const darkBackground = {
+  default: "Dark",
+  values: [
+    {
+      name: "Dark",
+      value: colors["blk-210"]
+    }
+  ]
+};
```

Comment:
Instead of initializing stories with dark/light backgrounds we could just have both backgrounds in the top toolbar for use at any time.

Reply:
Nice, I think I read in a "coming soon in Storybook 6" there was a way to use that toggle to then set a global context so switching the background would also set theme-as-state on a component.

Reply:
@macandcheese do you know which calcite color variable the light background should use if any?

Reply:
Would be nice to include:
$ui-background: $blk-005;
$ui-foreground-1: $blk-000;

$ui-background-dark: $blk-210;
$ui-foreground-1-dark: $blk-200;

Since those are really the possible background colors. We can look into setting global context via prop on the components later... first two would set theme to light, latter to dark.